src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -131,7 +130,11 @@ export class ExamUpdateComponent implements OnInit {
     get isValidExamStudentReviewEnd(): boolean {
         // allow instructors to set examStudentReviewEnd later
         if (!this.exam.examStudentReviewEnd) {
-            return true;
+            if (!this.exam.examStudentReviewStart || !this.exam.examStudentReviewStart.isValid()) {
+                return true;
+            } else {
+                return false;
```

Comment:
What do you think of returning the result inline, i.e.
```suggestion
            return !this.exam.examStudentReviewStart || !this.exam.examStudentReviewStart.isValid()
```

Reply:
You are right, this would make it easier to understand.